src/test/scala/io/iohk/ethereum/blockchain/sync/regular/RegularSyncSpec.scala

```diff
@@ -86,6 +86,8 @@ class RegularSyncSpec
   def sync[T <: Fixture](test: => T): Future[Assertion] =
     Future {
       test
+      // this makes sure that actors are all done after the test (believe me, afterEach does not work with mocks)
+      TestKit.shutdownActorSystem(testSystem)
```

Comment:
This 'fixture' pretty weird... are there any other tests with this?

Reply:
it's custom to this test, haven't seen it any other place.